fuzz.go

```diff
@@ -277,6 +274,16 @@ func intAddMod(f1, f2, f3, f4 *Int) *Int {
 	return f1.AddMod(f2, f3, f4)
 }
 
+func bigMulDiv(b1, b2, b3, b4 *big.Int) *big.Int {
+	b1.Mul(b2, b3)
+	return b1.Div(b1, b4)
+}
+
+func intMulDiv(f1, f2, f3, f4 *Int) *Int {
```

Comment:
This may be upgraded to public function in future so maybe change it to `func (z *Int) mulDiv(x, y, d *Int) *Int`?

Reply:
Let's do that later in that case